app/assets/stylesheets/application.scss

```diff
@@ -94,13 +86,15 @@ a {
 }
 
 a:link {
-color: #FF3300;
+  color: #FF3300;
```

Comment:
Color literals like `#FF3300` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FF3300` should be written as `#ff3300`